build_docker_image.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -x
```

Comment:
Can we move this to deployment

Reply:
The "deployment" directory is actually the /kb/deployment directory in the image, and these don't belong in the final built image. If you like I can put it into a directory where you keep administrative scripts related to the build, but I don't see one currently.

Reply:
There isn't one. How about dockerbuild or something?

Reply:
Maybe just "build" and any other build related tools can go in there?

Reply:
Sure

Reply:
Done.

Reply:
Looks like there's 2 copies of this file now.